package.json

```
@@ -37,19 +37,23 @@
     "@edx/brand": "npm:@edx/brand-openedx@1.1.0",
     "@edx/frontend-platform": "4.5.1",
     "@edx/paragon": "20.45.0",
+    "@emotion/react": "11.13.0",
+    "@emotion/styled": "11.13.0",
     "@fortawesome/fontawesome-free": "^6.5.2",
     "@fortawesome/fontawesome-svg-core": "1.2.36",
     "@fortawesome/free-brands-svg-icons": "5.15.4",
     "@fortawesome/free-regular-svg-icons": "5.15.4",
     "@fortawesome/free-solid-svg-icons": "5.15.4",
     "@fortawesome/react-fontawesome": "0.2.0",
+    "@mui/material": "5.16.6",
+    "@mui/x-charts": "7.12.0",
     "core-js": "3.25.5",
     "jquery": "1.9.1",
     "jsoneditor": "^10.1.0",
     "popper.js": "1.16.1",
     "prop-types": "15.8.1",
-    "react": "16.14.0",
-    "react-dom": "16.14.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
```

```suggestion
    "react": "17.0.2",
    "react-dom": "17.0.2",
```

Please do not forget to update the package-lock.json as well


Are you 100% sure this change does not broke anything? Like Authentication or any different stuff of open edx

---

@01001110J At least within the MFE everything seems to work well.